packages/captp/test/worker.js

```diff
@@ -0,0 +1,23 @@
+import '@agoric/install-ses';
+
+import { parentPort } from 'worker_threads';
+import { makeGuest, makeHost } from './traplib';
+
+let dispatch;
+parentPort.addListener('message', obj => {
+  switch (obj.type) {
+    case 'TEST_INIT': {
+      const { sab, isGuest } = obj;
+      const initFn = isGuest ? makeGuest : makeHost;
+      const ret = initFn(o => parentPort.postMessage(o), sab);
```

Comment:
I hadn't realized till here that `makeGuest` and `makeHost` were built to be somewhat interchangeable. Cool.

Reply:
Resolving.